### PR TITLE
Customer front end

### PR DESCRIPTION
**Pull request by scotteggs**

### CODE REVIEW

**Comment by scotteggs** [comment]
closes #33 closes #32

File: browser/js/common/factories/ProductFactory.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+app.factory('ProductFactory', function ($http) {
+	var ProductFactory = {};
+
+	ProductFactory.fetchAll = function () {
+		return $http.get('/api/product/')
+		.then(function (response) {
+			return response.data;
+		})
+		.catch(function(err) {
+			console.log(err);
```
Comment (danielsomekh) [inline browser/js/common/factories/ProductFactory.js:10]: can write function function getData(response){return response.data})

File: browser/js/common/factories/ProductFactory.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+		})
+	}
+
+	ProductFactory.fetchOne = function(id) {
+		return $http.get('/api/product/' + id)
+		.then(function (response) {
+			return response.data;
+		})
+		.catch(function(err) {
+			console.log(err);
```
Comment (danielsomekh) [inline browser/js/common/factories/ProductFactory.js:20]: same for log errror

File: browser/js/common/factories/UserFactory.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+app.factory('UserFactory', function ($http) {
+	var UserFactory = {};
+
```
Comment (danielsomekh) [inline browser/js/common/factories/UserFactory.js:3]: getData function

**Pull request merged**